server/api/versions/index.get.ts

```diff
@@ -0,0 +1,28 @@
+import { fetchRApi } from "@/server/utils/rApi";
+import type { RApiResponse } from "@/server/utils/rApi";
+import packageJson from "@/package.json";
+
+const rApiVersionEndpoint = "/";
+
+interface VersionDataResponse extends RApiResponse {
+  data: {
+    "daedalus": string
+    "daedalus.api": string
+  } | null
+}
+
+interface SuccessResponse {
+  daedalusModel: string
+  daedalusApi: string
+  daedalusWebApp: string
+}
+
+export default defineCachedEventHandler(async (event): Promise<SuccessResponse> => {
+  const response = await fetchRApi<VersionDataResponse>(rApiVersionEndpoint, {}, event);
+
+  return {
+    daedalusModel: response!.data!.daedalus,
+    daedalusApi: response!.data!["daedalus.api"],
+    daedalusWebApp: packageJson.version,
+  };
+}, { maxAge: 60 /* Cache response server-side for 1 minute */ }); // https://nitro.unjs.io/guide/cache
```

Comment:
This feels like it ought to be defined as a global setting somewhere, for responses which are appropriate for caching.

Reply:
60 in particular? I think the appropriate cache duration might vary by endpoint. It feels premature to make that a shared setting, until we have at least two cached endpoints.

Reply:
I ended up getting rid of the cacheing because it was causing confusion while debugging.